.travis.yml

```diff
@@ -7,6 +7,7 @@ before_install:
 - export GCLOUD_PROJECT=`cat tests/configuration/stub_project_id.txt`
 - export STUBBED_API_KEY=`cat tests/configuration/stub_api_key.txt`
 - export GOOGLE_APPLICATION_CREDENTIALS=$(pwd)/tests/configuration/stub_default_credentials.json
+- alias istanbul="./node_modules/.bin/istanbul"
```

Comment:
This should be removed, it doesn't appear to solve the problem. I'm not sure how travis/npm run scripts so it may be safer to move the script logic (istanbul/coverage calls) into separate scripts the trace and debug agents are still building and they call istanbul from separate bash scripts.